lib/Github/Api/Repository/Statuses.php

```diff
@@ -52,7 +52,7 @@ public function combined($username, $repository, $sha)
      *
      * @return array
      */
-    public function create($username, $repository, $sha, array $params = [])
```

Comment:
👎 Better to get the missing param exception than a fatal error.

Reply:
It's consistent with the rest of the code, though. So I feel like it should either be like this everywhere, where functions that make no sense without params are required to have the `$params` parameter, or the `$params` parameter is optional everywhere.